MMETestHost/ViewController.m

```diff
@@ -0,0 +1,32 @@
+//
+//  ViewController.m
+//  MapboxMobileEvents
+//
+//  Created by Dane Miluski on 6/10/20.
+//  Copyright © 2020 Mapbox. All rights reserved.
+//
+
+#import "ViewController.h"
+
+@interface ViewController ()
+
+@end
+
+@implementation ViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+}
+
+/*
```

Comment:
Not sure where the style guide is but generally we don't commit commented out code